echopype/clean/api.py

```diff
@@ -40,14 +44,20 @@ def mask_transient_noise(
         Calibrated Sv data with depth data variable.
     func: str, default `nanmean`
         Pooling function used in the pooled Sv aggregation.
-    depth_bin : str, default `20`m
-        Pooling radius along ``depth`` in meters.
-    num_side_pings : int, default `50`
+    depth_bin : str, default `10m`
+        Pooling radius vertically along `depth`.
+    num_side_pings : int, default `25`
         Number of side pings to look at for the pooling.
     exclude_above : Union[int, float], default `250`m
         Exclude all depth above (closer to the surface than) this value.
     transient_noise_threshold : Union[int, float], default `10.0`dB
         Transient noise threshold value (dB) for the pooling comparison.
+    use_index_binning : bool, default `False`
+        Speeds up aggregations by assuming depth is uniform and binning based
+        on `range_sample` indices instead of `depth` values.
+    chunk_dict : dict, default `{}`
+        Dictionary containing chunk sizes for use in the Dask-Image
+        pooling function. Only used when `use_index_binning=True`.
```

Comment:
How about adding an Example section in the docstring to show how a single call would look like?

Reply:
I'll add that